src/widgets/overview.js

```diff
@@ -13,75 +13,27 @@ import { REPORT_STATUSES } from '../constants';
   If adding a new widget be sure to add the widget to ./index.js
 */
 export default async function overview(scopes, query) {
```

Comment:
No action on this PR. Just getting this down for discussion.

I think this widget needs to be updated much in the same way `dashboardOverview` is being updated in https://github.com/adhocteam/Head-Start-TTADP/pull/376. Instead of having subqueries inside the select I think having completely separate SQL calls and stitching them together makes more sense. It seems easier to understand to me, scopes will be honored without worrying about the subquery and we probably don't add too much latency to the request because the subqueries are just moving out to their own call, instead of being ran inline. Doing two queries allows the overview widget to no longer care about the query, keeping the interface the same as other widgets. These widgets should not have to care about what is inside the `scope` parameter.

Another idea; we may want to combine the overview and dashboard overview widgets and control the fields shown with a prop on the frontend.

Reply:
This is great. I added this to a related ticket. https://ocio-jira.acf.hhs.gov/browse/TTAHUB-267